third_party/fmtlib.BUILD

```diff
@@ -0,0 +1,16 @@
+# Taken from https://github.com/envoyproxy/envoy/blob/f066fdc261429a1a00d2073fd76fd4d4f58f618e/bazel/external/fmtlib.BUILD
```

Comment:
Maybe we need a PSC policy to make it easier for reviewers to know what license applies to copied code.

Though this particular segment of Bazel code seems pretty standard issue for anybody who wants to just `build this glob of files`.

Reply:
PSC? :rofl:

Reply:
Oops, TSC =D Technical Steering Committee (Too much PSC on the brain)

Reply:
For the third party BUILD files added in this PR, I've simplified them to just be the vanilla include everything + build everything configuration. Now that they are pretty straightforward, I don't think any extra third party licensing comment is necessary here. But this convo still applies to any other BUILD files we may need to borrow from other repos in the future.